.github/workflows/release.yml

```
@@ -21,11 +21,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)